Yosemite/YosemiteTests/Stores/ShippingLabelStoreTests.swift

```diff
@@ -38,6 +38,130 @@ final class ShippingLabelStoreTests: XCTestCase {
+                                                                  country: "US",
+                                                                  state: "CA",
+                                                                  address1: "9999 19TH AVE",
+                                                                  address2: "",
+                                                                  city: "SAN FRANCISCO",
+                                                                  postcode: "94121-2303")
+        let expectedDestinationAddress = Yosemite.ShippingLabelAddress(company: "",
+                                                                       name: "Woo buyer",
+                                                                       phone: "1650345689",
+                                                                       country: "TW",
+                                                                       state: "Taiwan",
+                                                                       address1: "No 70 RA St",
+                                                                       address2: "",
+                                                                       city: "Taipei",
+                                                                       postcode: "100")
+        let expectedRefund = Yosemite.ShippingLabelRefund(dateRequested: Date(timeIntervalSince1970: 1603716266.809), status: .pending)
+        let expectedShippingLabel = Yosemite.ShippingLabel(siteID: sampleSiteID,
+                                                           orderID: orderID,
+                                                           shippingLabelID: 1149,
+                                                           carrierID: "usps",
+                                                           dateCreated: Date(timeIntervalSince1970: 1603716274.809),
+                                                           packageName: "box",
+                                                           rate: 58.81,
+                                                           currency: "USD",
+                                                           trackingNumber: "CM199912222US",
+                                                           serviceName: "USPS - Priority Mail International",
+                                                           refundableAmount: 58.81,
+                                                           status: .purchased,
+                                                           refund: expectedRefund,
+                                                           originAddress: expectedOriginAddress,
+                                                           destinationAddress: expectedDestinationAddress,
+                                                           productIDs: [3013],
+                                                           productNames: ["Password protected!"])
```

Comment:
This is very nitpicky. With the length, it gets a bit difficult to understand which variables are ultimately used or not. 

It might be worth it to encapsulate `expectedOriginAddress`, `expectedDestinationAddress` and `expectedRefund` into a `ShippingLabel` builder function to clarify that they're just helper variables. 

```suggestion
        let expectedShippingLabel: Yosemite.ShippingLabel = {
            let origin = ShippingLabelAddress(company: "fun testing",
                                              name: "Woo seller",
                                              phone: "6501234567",
                                              country: "US",
                                              state: "CA",
                                              address1: "9999 19TH AVE",
                                              address2: "",
                                              city: "SAN FRANCISCO",
                                              postcode: "94121-2303")
            let destination = ShippingLabelAddress(company: "",
                                                   name: "Woo buyer",
                                                   phone: "1650345689",
                                                   country: "TW",
                                                   state: "Taiwan",
                                                   address1: "No 70 RA St",
                                                   address2: "",
                                                   city: "Taipei",
                                                   postcode: "100")
            let refund = ShippingLabelRefund(dateRequested: Date(timeIntervalSince1970: 1603716266.809), status: .pending)

            return ShippingLabel(siteID: sampleSiteID,
                                 orderID: orderID,
                                 shippingLabelID: 1149,
                                 carrierID: "usps",
                                 dateCreated: Date(timeIntervalSince1970: 1603716274.809),
                                 packageName: "box",
                                 rate: 58.81,
                                 currency: "USD",
                                 trackingNumber: "CM199912222US",
                                 serviceName: "USPS - Priority Mail International",
                                 refundableAmount: 58.81,
                                 status: .purchased,
                                 refund: refund,
                                 originAddress: origin,
                                 destinationAddress: destination,
                                 productIDs: [3013],
                                 productNames: ["Password protected!"])

        }()
```

Reply:
Looks much nicer! Updated here https://github.com/woocommerce/woocommerce-ios/pull/3176/commits/344ec72aaff55eb55026eacd893be18dbc55aa2d